packages/core/src/lib/input/slider-input.svelte

```diff
@@ -114,13 +113,6 @@ $: handlePointerMove = (event: PointerEvent) => {
     value = next;
     dispatch('input', value);
   }
-
-  /**
-   * TODO: Determine why this is being interpreted as an `any` type by the
-   * linter when it is of `() => Promise<void>`.
```

Comment:
This appears to be a limitation of the Svelte ESLint plugin; it's a little buggy when it comes to correctly manipulating the parse tree when TS is involved (e.g. https://github.com/sveltejs/eslint-plugin-svelte/issues/583).

I've been moving type exports to separate pure TS files to work around the issue. It's a little irritating to move the interfaces farther away from the component that uses them; but it's worth it to avoid losing type safety